lib/slack.ex

```diff
@@ -49,13 +49,16 @@ defmodule Slack do
   """
   use HTTPoison.Base
 
-  @type response :: {:ok, map} | {:error, HTTPoison.Response.t} | {:error, HTTPoison.Error.t}
+  @type slack_response ::
```

Comment:
References to this type need to be updated elsewhere.

Reply:
Ah, yes, the reason for that change is because the latest version of HTTPoison also defines a `response` type and because that module `use`s it, they were tripping over each other. I'll fix up the other references.

Reply:
Of course the fact that a different `response` is defined elsewhere is exactly why the compiler didn't complain that I'd missed changing all the uses of it. That's fixed now - thanks.